src/app/pages/dashboard/dashboard.page.scss

```diff
@@ -0,0 +1,107 @@
+
```

Comment:
should we not be encapsulating these stylings to ensure that this only applies to the `<dashboard>` element? E.g. in the `mes-mobile-app` everything is enclosed in the following rule -
```
page-dashboard {
  // all styles go here
}
```

Reply:
The dashboard component specifies what style file to use (In component tag) so this shouldn't be necessary to my understanding, only in global styles we need to be careful

Reply:
Forgot that Angular did that for you - https://angular.io/guide/view-encapsulation